doc/tor.md

```diff
@@ -211,3 +209,105 @@ for normal IPv4/IPv6 communication, use:
   Otherwise it is trivial to link them, which may reduce privacy. Onion
   services created automatically (as in section 2) always have only one port
   open.
+
+- In some regions of the world usage of the Tor network is monitored and/or blocked.
+  If you are in an environment that does not permit direct Tor connections or the use
+  of Tor bridges, then considering the trade-offs, it may not be safe to use Tor.
+
+- For maximum privacy, it is preferable to disable accepting incoming connections.
+
+- Users can download packages with [torsocks](https://gitlab.torproject.org/legacy/trac/-/wikis/doc/torsocks) for better security and privacy.
+
+- Trade-offs involved in using `onlynet=onion`:
+
+  The `onlynet=onion` configuration option can potentially ensure the node only attempts to connect over
+  the Tor network. The decision to use this option should consider the following trade-offs:
```

Comment:
The first sentence is repeating the `onlynet` docs, but less precisely. Suggest dropping this paragraph completely. Brevity is good.

Reply:
> The first sentence is repeating the onlynet docs

Which docs?

> The second sentence repeats the bullet point.

Removed it.

Reply:
> > The first sentence is repeating the onlynet docs
> 
> Which docs?
 
The description in the list of options.

Reply:
Made changes in https://github.com/bitcoin/bitcoin/commit/cb46911a3b09f126eeb40e02234d8ca57757e3d6